mixins/StoreMixin.js

```diff
@@ -76,9 +76,16 @@ var StoreMixin = {
   store_addListeners: function () {
     Object.keys(this.store_listeners).forEach(function (storeID) {
       var listenerDetail = this.store_listeners[storeID];
+      var events = listenerDetail.events;
+      var eventKeys = Object.keys(events);
 
+      // Check that we actually have events to fire events on
+      if (!events || !eventKeys.length) {
```

Comment:
lunoe fixed it by using the storeID from the arguments.

IMO we should actually test that `this.store_listeners` even exists and throw an error if it doesn't